### ISSUE TITLE
Strava Sync: Please continue your synchronization in 4~5 hours:

### ISSUE THREAD

**Issue by johnny-str** [thomaschampagne/elevate#835]
Hi,
Do you have any suggestions for overcoming this limitation?
I have 1950 activities in strava and could never synchronize, always get this message.
thanks

![image](https://user-images.githubusercontent.com/17263090/58341088-537c2980-7e45-11e9-8557-bb9f4493715b.png)

**Comment by scadogan** [comment]
I am having the exact same issue as i have recently moved to a new computer.  I have waited the requisite 4 - 5 hours but still receive the same error.  I also receive an error Unable to generate fitness trend, but there is nothing in the setting that require changing.  The sync works fine on my old PC.
Cheers Sean

**Comment by brendonrjones** [comment]
I'm having the same issue. I'm using Elevate already and I'm trying to use a separate tab in Chrome to sync my wife's data (separate account).  She has 569 activities and it will process 400 of them before it comes up with that 'error' message. I've waited and tried again to no avail.

Cheers,
Brendon

**Comment by rvhilst** [comment]
Any resolution to this one peeps? After waiting overnight, I get the error message on the first attempt to re-sync

**Comment by thomaschampagne** [comment]
Unfortunately you have to sync ~400 activities, and restart later to sync ~400 activities more, etc...
It's Strava server limitation they introduced in February. Message displayed is quite clear, it's not a bug or an error.

However, I am working on something new since February that will no longer be strongly related to Strava as it is today. (That's why updates frequency on web plugin is lower). 

Just need time... I do that on my tiny free time :/

**Comment by thomaschampagne** [comment]
> I'm having the same issue. I'm using Elevate already and I'm trying to use a separate tab in Chrome to sync my wife's data (separate account). She has 569 activities and it will process 400 of them before it comes up with that 'error' message. I've waited and tried again to no avail.
> 
> Cheers,
> Brendon

You should use separate chrome accounts: https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions#multiple-strava-accounts--data-collision

**Comment by thomaschampagne** [comment]
> I am having the exact same issue as i have recently moved to a new computer. I have waited the requisite 4 - 5 hours but still receive the same error. I also receive an error Unable to generate fitness trend, but there is nothing in the setting that require changing. The sync works fine on my old PC.
> Cheers Sean

Then backup elevate data from old computer and import to new one to bypass the first painfull sync ;)

**Comment by rvhilst** [comment]
Thanks for the reply Thomas. I had left it overnight and restarted, but got
the error message on my first attempt. I'll wait another day and try again

Cheers
Roger

**Comment by PhoiG** [comment]
Obviously this can still happen, when there's a corrupted file on Strava. 
I tried it a few times (with 4-10 hours in between). Sync started again at 0/1513 and stopped at 580 - but from second try on it needed less than 30 seconds. (So obviously just scanning existing files.)
But then I discovered file 580 was a Zwift ride which didn't save properly, so obviously something wrong with that file. I deleted it and at the moment sync is at 720. :-)
